src/Sylius/Component/Pricing/Model/PriceableInterface.php

```diff
@@ -33,6 +33,20 @@ public function getPrice();
     public function setPrice($price);
 
     /**
+     * Get original price.
+     *
+     * @return integer
+     */
+    public function getOriginalPrice();
+
+    /**
+     * Set original price.
+     *
+     * @param integer $originalPrice
+     */
+    public function setOriginalPrice($originalPrice);
```

Comment:
I think these methods should go to core interface, original price does not affect any of the logic in Pricing component.